### DISCUSSION TITLE
Small UI improvement: swap Batch Size and Batch Count positions in img2img settings

### DISCUSSION THREAD

**Discussion by eremtsov** [hlky/stable-diffusion-webui#249] [Feature suggestions]
![image](https://user-images.githubusercontent.com/34932866/187161465-87dd1d04-fd98-4bfa-a7bf-429c9b294ebe.png)
Since Batch Size is more memory hungry than Batch Count and they basically do the same thing (generate more images), I think it's more preferable to use Batch Count and move it to the top. And why do we even have Batch Size if it's the same thing but worse?

**Comment by eremtsov** [comment]
I think it would also make sense to make the same order of settings in txt and img modes for consistency:
![image](https://user-images.githubusercontent.com/34932866/187162340-14bcda72-af9c-45c3-bd90-d8a62b8251ef.png)

**Comment by jansteffen** [comment]
A lot of stuff in the img2img tab got moved around during yesterdays UI refactor, we'll definitely have to redo the layout.

I also question the usefulness of batch size, but I'll leave that up to @hlky

**Comment by ZeroCool940711** [comment]
@jansteffen if im not wrong I think batch size generate a batch of images at once while Batch count do it one after the other, thats why batch size uses more memory while batch count just increases the number of images but still uses the same memory no matter how many images you create and it only increases the total time it will take to generate everything as there are more images to generate in total, also, batch size is a lot useful for people with extra VRAM to spare as they can speed the generation a lot, by combining both options you can get a lot of images in  short amount of time if you have enough memory for that.

**Comment by hlky** [comment]
![image](https://user-images.githubusercontent.com/106811348/187190787-e39af6f4-8e2e-4fab-8971-ab120f2d1121.png)

**Comment by jansteffen** [comment]
I've gone ahead and created a new branch with a first set of small changes https://github.com/hlky/stable-diffusion-webui/tree/img2img_layout_improvements

![image](https://user-images.githubusercontent.com/1278331/187197563-5e87b157-7ce9-41f7-bbbb-06b02430fe00.png)

Unfortunately I'm quite busy for the rest of the day but I'd love to see suggestions from others.

**Comment by gmaxwell** [comment]
The latest updates seem to have removed batch count entirely from img2img, entirely.  This is unfortunate for me, since I have enough gpu ram to do 2 image batches at 512x512 without issue and get about a 20% throughput increase from doing so.

On text-to-image it's under advanced, which is a fine place for it.
